### DISCUSSION TITLE
How do I disable Rich markup interpretation on a Widget?

### DISCUSSION THREAD

**Discussion by TheBigS** [Textualize/textual#169] [Q&A] [answered]
I've got a panel that I want to render debug text to so I don't want any styles or Rich markup interpretations on the contents. So if my debug output contains the literal string `"[bold]"` then I don't want it to start bolding the output. I tried initializing the `Console(markup=False)` and passing to `MyApp()` but that's not working. 

My preference would be to disable markup interpretation on just one widget rather than the entire app anyway.
Anyone know how disable Rich markup interpretation on a Widget?

**Comment by willmcgugan** [comment, marked as answer]
You can [escape console markup](https://rich.readthedocs.io/en/latest/markup.html), or work with Text objects which won't implicitly render markup

**Reply by TheBigS** [reply]
Thanks, I was really close. I just changed my widget render to this and it is working like I want it now!

`return Lines([Text(c) for c in content.split("\n")])`
